lib/Config/RawConfig.cpp

```diff
@@ -14,6 +16,14 @@ int MullDefaultTimeoutMilliseconds = 3000;
 
 using namespace mull;
 
+static std::string getFileDir(const std::string &filePath) {
+  assert(filePath.size() < 256 &&
+         "Expected file path string to be of realistic size");
+  char path[256];
+  strncpy(path, filePath.c_str(), filePath.size() + 1); // + 1 goes for '\0'
+  return std::string(path);
+}
```

Comment:
Something is missing here, right?

Reply:
👍

Reply:
I am not yet testing those paths, only the default fallback to `/tmp/....json` file.

Reply:
Actually, do you think it is a good idea to extract this function to some FileUtils and write a unit test for it?

P.S. I could not find anything in `llvm::sys::fs`.

Reply:
`mull::file_utils`?

Reply:
I have very strong objections against `*Utils` :)

Reply:
Then just follow the approach of llvm:

`mull::fs` or `mull::sys::fs`...

The point is to have it in a dedicated place with some other helpers that we had. Also with some tests.

Reply:
Do we have other helpers? I don't mind having a separate place, but not with one function

Reply:
You can look into the latest version of LLVM, if there is such function, then we can backport it under `llvm_compat` namespace.

Reply:
I found nothing relevant in the latest llvm::sys::fs docs.